src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -846,7 +847,14 @@ export default class GeFormRenderer extends NavigationMixin(LightningElement) {
             let displayValue;
 
             if (isDonorLookupAndHasValue) {
-                displayValue = this.selectedDonation.Name;
+                if (fieldApiName === donationImported) {
+                    displayValue = this.selectedDonation.Name;
+                }
+                if (fieldApiName === paymentImported) {
```

Comment:
I think I get the gist here, but just to make sure it's clear - what's the case this "if" block is handling?